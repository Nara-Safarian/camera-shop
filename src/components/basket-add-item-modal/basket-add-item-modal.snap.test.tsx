import { render } from '@testing-library/react';
import BasketAddItemModal from './basket-add-item-modal';

test('ChangeMe snapshot should be rendered correctly', () => {
  const {container} = render(
    <BasketAddItemModal isActive={false} onClose={function (): void {
      throw new Error('Function not implemented.');
    } } onAddProduct={function (id: number): void {
      throw new Error('Function not implemented.');
    } }
    />
  );
  expect(container).toMatchSnapshot();
});
