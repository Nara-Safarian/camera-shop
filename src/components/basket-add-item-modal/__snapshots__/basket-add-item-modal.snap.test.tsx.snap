// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChangeMe snapshot should be rendered correctly 1`] = `
<div>
  <div
    class="modal "
  >
    <div
      class="modal__wrapper"
    >
      <div
        class="modal__overlay"
      />
      <div
        class="modal__content"
      >
        <p
          class="title title--h4"
        >
          Добавить товар в корзину
        </p>
        <div
          class="basket-item basket-item--short"
        >
          <div
            class="basket-item__img"
          >
            <picture>
              <source
                srcset=",  2x"
                type="image/webp"
              />
              <img
                alt=""
                height="120"
                src=""
                srcset=" 2x"
                width="140"
              />
            </picture>
          </div>
          <div
            class="basket-item__description"
          >
            <p
              class="basket-item__title"
            />
            <ul
              class="basket-item__list"
            >
              <li
                class="basket-item__list-item"
              >
                <span
                  class="basket-item__article"
                >
                  Артикул:
                </span>
                 
                <span
                  class="basket-item__number"
                />
              </li>
              <li
                class="basket-item__list-item"
              >
                 
              </li>
              <li
                class="basket-item__list-item"
              >
                 уровень
              </li>
            </ul>
            <p
              class="basket-item__price"
            >
              <span
                class="visually-hidden"
              >
                Цена:
              </span>
               ₽
            </p>
          </div>
        </div>
        <div
          class="modal__buttons"
        >
          <button
            class="btn btn--purple modal__btn modal__btn--fit-width"
            type="button"
          >
            <svg
              aria-hidden="true"
              height="16"
              width="24"
            >
              <use
                xlink:href="#icon-add-basket"
              />
            </svg>
            Добавить в корзину
          </button>
        </div>
        <button
          aria-label="Закрыть попап"
          class="cross-btn"
          type="button"
        >
          <svg
            aria-hidden="true"
            height="10"
            width="10"
          >
            <use
              xlink:href="#icon-close"
            />
          </svg>
        </button>
      </div>
    </div>
  </div>
</div>
`;
